service/test/controllers/organisationmanagement/OrgTypeControllerTest.java

```diff
@@ -0,0 +1,113 @@
+package controllers.organisationmanagement;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import controllers.BaseControllerTest;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Test;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.responsecode.ResponseCode;
+import play.mvc.Result;
+import play.test.Helpers;
+
+public class OrgTypeControllerTest extends BaseControllerTest {
+
+  private static String orgTypeName = "org-type-name";
+  private static String id = "id";
+
+  @Test
+  public void testListOrgTypeListSuccess() {
+
+    Result result = performTest("/v1/org/type/list", "GET", null);
+    String response = Helpers.contentAsString(result);
+    assertTrue(response.contains("success"));
+    assertEquals(200, result.status());
+  }
+
+  @Test
+  public void testCreateOrgTypeSuccess() {
+
+    Result result =
+        performTest(
+            "/v1/org/type/create", "POST", createOrgTypeRequest(true, orgTypeName, false, null));
+
+    String response = Helpers.contentAsString(result);
+    assertTrue(response.contains("success"));
+    assertEquals(200, result.status());
+  }
+
+  @Test
+  public void testCreateOrgTypeFailureWithEmptyName() {
+
+    Result result =
+        performTest("/v1/org/type/create", "POST", createOrgTypeRequest(true, null, false, null));
+
+    String response = Helpers.contentAsString(result);
+    assertTrue(response.contains(ResponseCode.mandatoryParamsMissing.getErrorCode()));
```

Comment:
Don't check for expected response code anywhere in the response. You can add methods in BaseControllerTest for getResponseCode(result) and getResponse(result).